draft-ietf-idr-link-bandwidth.xml

```diff
@@ -127,27 +127,31 @@
       network performance. Traditional equal-cost multi-path (ECMP) routing
       does not account for the varying capacities of different paths. This
       document suggests that the external link bandwidth be carried in the
-      network using a new extended community <xref target="RFC4360"/> - the
-      link bandwidth extended community. The Link Bandwidth Extended Community
+      network using one of two new extended communities <xref target="RFC4360"/>
+      - the transitive and non-transitive link bandwidth extended community. 
+      The Link Bandwidth Extended Community
       provides a mechanism for routers to advertise the bandwidth of their
-      links, facilitating maximum utilisation of paths.</t>
+      downstream path(s), facilitating maximum utilisation of network resources.</t>
     </section>
 
     <section title="Link Bandwidth Extended Community">
-      <t>The Link Bandwidth Extended Community is defined as a BGP extended
-      community that carries the bandwidth information of a link connecting to
-      a neighbor. This community can be used to inform other routers about the
-      available bandwidth on a given link.</t>
-
-      <t>The extended community can be optional transitive or non-transitive.
-      The value of the high-order octet of the extended Type Field can be 0x00
-      or 0x40. The value of the low-order octet of the extended type field for
-      this community is 0x04. The value of the Global Administrator subfield
+      <t>The Link Bandwidth Extended Communities are defined as a BGP extended
+      community that carries the bandwidth information of a router, represented by
+      BGP Protocol Next Hop, connecting to remote network. 
+      This community can be used to inform other routers about the
+      available bandwidth on trough a given route.</t>
+
+      <t>Two extended communities are both optional, one transitive and other
```

Comment:
I think the phrase 'optional non-transitive' in draft-07 itself was misleading. we could remove the 'optional' altogether. since there is no concept of optional/mandatory for community/ext-comm.